lib/active_merchant/billing/gateways/mundipagg.rb

```diff
@@ -0,0 +1,288 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class MundipaggGateway < Gateway
+      self.test_url = 'https://api.mundipagg.com/core/v1/'
+      self.live_url = 'https://api.mundipagg.com/core/v1/'
```

Comment:
@nfarve - If the gateway doesn't provide a sandbox with a test URL, there's not much point in have both URLs. The `test_url` and conditional logic around the `live_url` can be removed with there's no sandbox. Thoughts?